### ISSUE TITLE
Admin script configuration UI: add possibility to enable/disable scheduling

### ISSUE THREAD

**Issue by bugy** [bugy/script-server#362]

**Comment by mitchtchesnitch** [comment]
Thought I'd leave my workaround here as well, since I've seen it mentioned in some threads, but no actual examples.

Simple script that checks the folder where the schedules are stored, giving you the option to "manage them", as in delete or list contents. Not pretty but does the job:

![image](https://user-images.githubusercontent.com/71206843/97975886-7ba9df00-1dd2-11eb-8258-ebd17b65ff44.png)

Runner: 

`{
    "name": "\ud83d\udcbbAdmin Schedule Management",
    "script_path": "/Script-Server/conf/scripts/Scheduled/admin_schedule_management.sh",
    "working_directory": "/",
    "description": "Script used for administering schedules",
    "group": "Administration",
    "requires_terminal": true,
    "allowed_users": [ "@blabla"],
    "parameters": [
      {
        "name": "Action",
        "required": true,
        "type": "list",
        "description": "Which action would you like to perform?",
        "default": "list",
        "values": [
            "list",
            "delete"
        ]
      },
      {
        "name": "File",
        "required": true,
        "type": "server_file",
        "description": "Which file do you want to work on?",
        "file_dir": "/Script-Server/conf/schedules",
        "file_recursive": "True",
        "file_type": "file"
      }
    ]
}`


Script:


`#!/bin/sh

ACTION=${1}
FILE_RAW=${2}
FILE=$(echo $FILE_RAW | sed 's/ /\\ /g') #Strips spaces from filenames, posix doesn't like that.



case ${ACTION} in
    *list*)
    COMMAND="cat"
    ;;
    *delete*)
    COMMAND="rm"
    ;;
    *)
    printf "$red" "Invalid option, please chose list/delete"
esac

printf "Okay, running ${COMMAND} on the selected file: ${FILE}"
printf "\n\n"
${COMMAND} ${FILE}`

**Comment by bugy** [comment]
Hehe, nice! :)

One side note question out of curiosity: why are you using `"file_recursive": "True"`? It's not necessary, when you don't have nested folders

**Comment by mitchtchesnitch** [comment]
You're right, now that I think of it, that doesn't make sense, I removed that:D

**Comment by Gamma-Software** [comment]
I replaced ${COMMAND} ${FILE}` by ${COMMAND} "${FILE}"`
Reason: I was not able to remove schedules configs with space in names

**Comment by bugy** [comment]
just a note: this issue is about adding a possibility on admin UI to enable scheduling for a particular script configuration:
![image](https://user-images.githubusercontent.com/1275813/219694562-0fd558cd-2c59-478b-a7e8-4dbdecd1cdbd.png)

And this is resolved now

There is a separate ticket in #625 for managing created schedules
